experimental/framework/tinylicious-client/src/index.ts

```diff
@@ -0,0 +1,11 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { FluidTinylicious } from "./FluidStaticTinylicious";
+export * from "./tinyliciousService";
+
+export { InsecureTinyliciousUrlResolver } from "@fluidframework/tinylicious-driver";
```

Comment:
When does the developer need this?

Reply:
This was kept in parity with get-container but if the developer is using the TinyliciousClient directly, they should not have to directly interact with the url resolver or build their own loader. Removed.